actions.yaml

```diff
@@ -43,6 +43,26 @@ set-node-key:
       type: string
   required: [ key ]
 
+find-validator-address:
+  description: |
+    Checks if this node is currently validating for any validator address found on-chain.
+    It does so by checking if any session key on-chain is present on this node.
+    Returns the validator address and the session key if this node is validating.
+
+is-validating-this-era:
+  description: |
+    Checks if this node is currently validating for the validator address set in config.
+    It does so by checking if the session key used for that validator is present on this node.
+    Returns a message whether this node is validating or not. If it is validating, the session key is returned as well.
+    REQUIRES validator-address config parameter to be set.
+
+is-validating-next-era:
+  description: |
+    Checks if this node will be validating next era for the validator address set in config.
+    It does so by checking if the session key that will be used for that validator is present on this node.
+    Returns a message whether this node will be validating next era or not. If it will, the session key is returned as well.
+    REQUIRES validator-address config parameter to be set.
+
```

Comment:
I have been thinking about this a lot.
1. It would be possible for `is-validating-this-era` since it uses the same RPC method as `find-validator-address`. But one drawback is that without the address the time complexity increases a lot. It needs to check every validator existing on-chain if the session key exist on the node. While already having the address it can look for that validator directly. And since it is used in the `update-status` hook it is nice if it is more lightweight.
2. `is-validating-next-era` uses another RPC methods that requires the address as parameter. So not possible without knowing the address.